templates/stub.erb

```diff
@@ -1,5 +1,11 @@
 stub-zone:
   name: "<%= @name %>"
+<% if @address -%>
 <% Array(@address).each do |addr| -%>
+<% if /\A\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}\Z/ =~ addr -%>
```

Comment:
If this is meant to validate an IP address, I'd prefer that this get moved outside of the template.  Also, this test should take into consideration IPv6.